<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectedEventDialog</class>
 <widget class="QDialog" name="SelectedEventDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/resource/icons/calendar icon.svg</normaloff>:/resource/icons/calendar icon.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="1" column="1">
          <widget class="QLineEdit" name="eventname_input"/>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="about_input"/>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QCheckBox" name="sendmessage_checkbox">
           <property name="text">
            <string>Send Message</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="to_label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>To</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="to_input"/>
         </item>
         <item row="9" column="1">
          <widget class="QLineEdit" name="subject_input"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="eventname_label">
           <property name="text">
            <string>Event Name *</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="about_label">
           <property name="text">
            <string>About Event *</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="subject_label">
           <property name="text">
            <string>Subject          </string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="message_label">
           <property name="text">
            <string>Message</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="message_input"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="modify_button">
         <property name="text">
          <string>Modify</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="close_button">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="delete_button">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>eventname_input</tabstop>
  <tabstop>about_input</tabstop>
  <tabstop>sendmessage_checkbox</tabstop>
  <tabstop>to_input</tabstop>
  <tabstop>subject_input</tabstop>
  <tabstop>message_input</tabstop>
  <tabstop>modify_button</tabstop>
  <tabstop>close_button</tabstop>
  <tabstop>delete_button</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
